website_event_attendee_signup/static/description/index.html

```diff
@@ -0,0 +1,93 @@
+<div class="container">
+            <div class="oe_styling_v8">
+				<section class="oe_container">
+					<div class="oe_row oe_spaced">
+						<h2 class="oe_slogan" style="color:#875A7B;">Sign-up event attendees to portal</h2>
+						<h3 class="oe_slogan">Attendees can use portal dashboard to make extra purchases for the event, for example.</h3>
+					</div>
+				</section>
+
+				<section class="oe_container oe_dark">
+				  <div class="oe_row oe_spaced">
+					<div class="oe_span12">
+					  <div class="alert alert-warning" style="padding:4px;font-weight: 300; font-size: 20px;">
+						<p>
+						   The modules creates User from every Event registration send an email with signup url to access the portal.
+						</p>
+						<p class="oe_mt32">
+						  The modules adds email template Event: Signup, which can be used directly or as an example to modify other email template.
+						</p>
+					  </div>
+					</div>
+				  </div>
+				</section>
+
+				<section class="oe_container">
+				  <div class="oe_row oe_spaced">
+					<h3 class="oe_slogan">How It Works</h3>
+					<h5 class="oe_slogan" style="font-size:16px;">
+								You can install this module in a <a title="How to install/update module" href="https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html">usual way</a>. After installation, you need to configure the module according to the documentation.
+					</h5>
+					<div class="oe_span12 text-center">
+					  <p class="oe_mt32">
+						<font size="3px">Register few attendees to the Event via Website (/event page).<font>
+					  </font></font></p>
+					  <p class="oe_mt32">
+						<font size="3px">Confirm the registrations (e.g. via backend).<font>
+					  </font></font></p>
+					  <p class="oe_mt32">
+						<font size="3px">As a result, each participant will receive an email with a link to register on the portal.<font>
+					  </font></font></p>
+					</div>
+					<div class="oe_row_img oe_centered">
+					  <img class="oe_demo oe_picture oe_screenshot" src="events-custom-fields-10.png">
+					</div>
+				   </div>
+				</section>
```

Comment:
I suggest to add a screenshot of email that user will receive after registration. 

The description has to show the idea, that if there several attendees in one checkout, then every person will receive the email to continue signup process

(You can get formatted email via Settings >> Email >> Email Templates menu)